static/app/views/sharedGroupDetails/sharedGroupHeader.tsx

```diff
@@ -33,6 +41,27 @@ function SharedGroupHeader({group}: Props) {
               />
             )}
           </ShortIdWrapper>
+
+          <TimeStamp data-test-id="sgh-timestamp">
+            {'Last seen '}
+            {(event.dateCreated ?? event.dateReceived) && (
+              <EventTimeLabel>
+                {getDynamicText({
+                  fixed: 'Jan 1, 12:00 AM',
+                  value: (
+                    <Tooltip
```

Comment:
we don't need to use `getDynamicText` anymore, just use everything inside the value starting with <Tooltip